### ISSUE TITLE
fromGeoJson - add possibility to have polygon with different fill color, border width and border color

### ISSUE THREAD

**Issue by dariocavada** [synw/map_controller#4]
**addPolygon** has properties such as color, border width and border color. It would be convenient to have the ability to set these properties when loading a **GeoJson**. 
For example for Indoor Maps, have different colors for walls and furniture.

**Comment by synw** [comment]
Good idea. We can implement this from the geojson multipolygon properties
